cli/src/main/resources/hudson/cli/client/Messages.properties

```diff
@@ -11,10 +11,12 @@ CLI.Usage=Jenkins CLI\n\
   \ -user               : specify user (for use with -ssh)\n\
   \ -strictHostKey      : request strict host key checking (for use with -ssh)\n\
   \ -logger FINE        : enable detailed logging from the client\n\
-  \ -auth [ USER:SECRET | @FILE ] : specify username and either password or API token (or load from them both from a file);\n\
-  \                                 for use with -http.\n\
-  \                                 Passing credentials by file is recommended.\n\
-  \                                 See https://jenkins.io/redirect/cli-http-connection-mode for more info and options.\n\
+  \ -auth [ USER:SECRET | TOKEN | @FILE ]\n\
+  \                     : specify username and either password or API token (or load from them both from a file);\n\
```

Comment:
```suggestion
  \                     : specify username and  password or API token (or load from them both from a file);\n\
```

Reply:
I think this changes the meaning. I added the next line about bearer token. This is about that you can provide Jenkins API token or Jenkins password, so I think it should be left as is.

Reply:
Okay thanks :+1: